jwst/regtest/test_nirspec_fs_spec2.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.lib.suffix import replace_suffix
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+file_roots = ['jw00023001001_01101_00001_nrs1_', 'jw93045010001_02101_00001_nrs2_', 'jwtest1013001_01101_00001_nrs1_']
+@pytest.fixture(scope="module", params=file_roots, ids=file_roots)
```

Comment:
I have learned that the specification of `ids` at this point is actually redundant; the parametrization accomplishes the same thing.

Reply:
Good to know. I just copy your code, under the assumption that you know waaaay more than I do about how stuff works. ;-)